Routes/admin/index.js

```diff
@@ -192,6 +193,36 @@ adminRouter.post('/sessions', (req, res) => {
   });
 });
 
+adminRouter.delete('/sessions/:id', (req, res) => {
+  Session.findByIdAndDelete(req.params.id, (err, data) => {
+    if (!data) {
+      return res.status(400).json({
```

Comment:
here you should return 404 because you are trying to access resource specified by url